tide/core/Content.h

```diff
@@ -105,6 +106,12 @@ class Content : public QObject
     /** Get the dimensions. */
     QSize getDimensions() const;
 
+    /** Get the width. */
+    int width() const;
```

Comment:
Your right that it is inconsistent, and I don't really plan to change all the naming conventions, although this looks nicer to use sometimes. The motivation for adding this was to be able to pass any object (Content, QRectF, QSize, DisplayGroup, ContentWindow..) to the templated functions in geometry.h. Rectangle.h is another exception where the get* prefix was already omitted. But of course, it is also possible to achieve the same other means, such as partial template specialization for instance.